intrastat_product/views/intrastat_product_declaration.xml

```diff
@@ -287,11 +294,17 @@
                     attrs="{'required': [('reporting_level', '=', 'extended')], 'invisible': [('reporting_level', '!=', 'extended')]}"
                 />
                 <field name="region_id" invisible="1" />
+                <!-- TODO: remove product_origin_country_id after update of localization modules -->
                 <field
                     name="product_origin_country_id"
                     invisible="1"
                     string="Product C/O"
                 />
+                <field
+                    name="product_origin_country_code"
+                    attrs="{'column_invisible': [('parent.declaration_type', '=', 'arrivals')]}"
+                    string="Product C/O"
```

Comment:
In France, since for reform of January 2022 1st, we have to declare the product country of origin on arrivals AND departures! I would be very surprised if it was not the case in other countries. Cf page 28 section "9. Pays d'origine" of this PDF : https://www.douane.gouv.fr/sites/default/files/2021-12/21/Note-enquete-statistique-EMEBI-2022.pdf